docs/kiwix-serve.rst

```diff
@@ -51,7 +51,11 @@ Options
 .. option:: -i ADDR, --address=ADDR
 
   Listen only on this IP address. By default the server listens on all
-  available IP addresses.
+  available IP addresses. Alternatively, you can use special values to define which types of connections to accept:
+
+  all : Accept connections from all IP addresses (IPv4 and IPv6).
+  ipv4 : Accept connections from all IPv4 addresses.
+  ipv6 : Accept connections from all IPv6 addresses.
```

Comment:
"Accept connections from an IP address" is very different from "Listen for connections on an IP address"